index.html

```diff
@@ -46,5 +41,176 @@ <h3 class="logo">John Doe </h3>
+            dummy text ever since the 1500s, when an 
+            unknown printer took a standard dummy 
+            text.</p>
+        <ul class="language">
+            <li class="css">css</li>
+            <li class="html">html</li>
+            <li class="bootstrap">bootstrap</li>
+            <li class="ruby">ruby</li>
+        </ul>
+        <button class="btnSeeProject">See Project</button>
+    </div>
+
+        <!--Section for the Card in multiple places-->
+    <div class="cards">
+        <div class="card">
+            <h4>Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard</p>
+    
+        <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    <div class="tags">
+        <ul>
+            <li><a href="">html</a></li>
+            <li><a href="">bootstrap</a></li>
+            <li><a href="">Ruby</a></li>
+        </ul>
+    </div>
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    <div class="tags">
+        <ul>
+            <li><a href="">html</a></li>
+            <li><a href="">bootstrap</a></li>
+            <li><a href="">Ruby</a></li>
+        </ul>
+    </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    
+        <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    
+        <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
```

Comment:
- [ ] As said by the previous reviewer, Kindly remove these anchors tags as they don't have any semantic meaning here.